packages/cozy-harvest-lib/src/components/TriggerLauncher.jsx

```diff
@@ -17,6 +17,7 @@ const TWO_FA_REQUEST = 'TWO_FA_REQUEST'
 const TWO_FA_MISMATCH = 'TWO_FA_MISMATCH'
 const PENDING = 'PENDING'
 const SUCCESS = 'SUCCESS'
+const RUNNING_TWOFA = 'RUNNING_TWOFA'
```

Comment:
`RUNNING_TWOFA` can be deduced with `TWO_FA_REQUEST`and `TWO_FA_MISMATCH` and should not exist, at least not at this level.

Letting konnectorJob manage statuses and relying on it may spare us listing those statuses here, as @ptbrowne suggests.

Reply:
Done in https://github.com/cozy/cozy-libs/pull/415/commits/fee5d4745535bc71fb6badaf248b2b3fcfb847d3